config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
   root 'home#index'
-  resources :users, only:[:edit, :update , :destroy]
   devise_for :users , path: 'session'
+  resources :users, only:[:edit, :update , :destroy]
```

Comment:
Atkal spaces.
Vai tu izlasiji to rakstu kuru metam čata? https://github.com/airbnb/ruby#indentation
Taču daudz lasamak būtu:
```ruby
resources :users, only: [:edit, :update, :destroy]

```